README.rst

```diff
@@ -124,5 +136,6 @@ Templates are in:
 * ``django_social_share/templatetags/send_email.html``
 * ``django_social_share/templatetags/post_to_linkedin.html``
 * ``django_social_share/templatetags/post_to_reddit.html``.
-  
+* ``django_social_share/templatetags/chama_no_zap_bb.html``.
```

Comment:
this is incorrect, it should be `django_social_share/templatetags/whatsapp.html`